Git-Squash.md

```diff
@@ -0,0 +1,22 @@
+# Squashing Multiple Commits Into One
+
+This is an awesome feature of `rebase` that can be used in the `interactive` mode. To squash the last _n_ commits into one, run the following command:
+
+```
+git rebase -i HEAD~n
+```
+
+That will open up a text-editor with something similar to the following:
+
+```
+pick commit_1
+pick commit_2
+pick commit_3
+...
+pick commit_n
+# Bunch of comments
+```
+
```

Comment:
Also add a line here suggesting to read the instructions given in the text editor at the bottom.